dags/variables.py

```diff
@@ -62,6 +66,23 @@ def read_load_dag_vars(var_prefix, **kwargs):
     return vars
 
 
+def read_load_dag_redshift_vars(var_prefix, **kwargs):
+    vars = {
+        'output_bucket': read_var('output_bucket', var_prefix, True, **kwargs),
+        'aws_access_key_id':  read_var('aws_access_key_id', var_prefix, True, **kwargs),
+        'aws_secret_access_key': read_var('aws_secret_access_key', var_prefix, True, **kwargs),
+        'notification_emails': read_var('notification_emails', None, False, **kwargs),
+        'schedule_interval': read_var('schedule_interval', var_prefix, True, **kwargs),
+    }
+
+    load_start_date = read_var('load_start_date', var_prefix, False, **kwargs)
+    if load_start_date is not None:
+        load_start_date = datetime.strptime(load_start_date, '%Y-%m-%d')
+        var_prefix['load_start_date'] = load_start_date
```

Comment:
I had a bug in my code. It should be `vars` instead of `var_prefix` here.

Reply:
@medvedev1088 - This is what you were referring to, right?

https://github.com/iter-io/ethereum-etl-airflow/commit/54120f414a6562b7a81be06e5ed112140f3c4134